### PR TITLE
Carousel banner addition

### PR DESCRIPTION
**Pull request by dev-enforced**
In this Pull Request, the reviewer will find the code required to develop the Home Page Section of MI STREAM.
Which has 3 major sections:
1) Snapshots of Mock Interviews compiled through an interactive slider
2) Introduction to the app and the why of developing and using it
3) Placeholder cards for category section that will be replaced in the upcoming versions!!

### CODE REVIEW

**Review by mariyasada — APPROVED** [review]
The code structure is good. and the carousel looks cool.good to go

File: src/App.module.css [hunk 1]
```diff
@@ -42,7 +42,7 @@
 .pageContent{
   display: flex;
   position: relative;
-  margin-top: 5rem;
+  /* margin-top: 5rem; */
```
Comment (shivanigangadharan) [inline src/App.module.css:45]: can remove this comment if not reqd

File: src/components/Categories/Categories.module.css [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+  text-transform: uppercase;
+  text-align: center;
+}
+.card_info {
+  padding: 15px;
+  margin: 10px 0;
+  border: 1px dashed #0071bc;
+}
+.card_info_title {
+  color: #0071bc;
```
Comment (shivanigangadharan) [inline src/components/Categories/Categories.module.css:39]: you might want to use a color variable here for title color.

File: src/components/HeroSection/HeroSection.jsx [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+        </p>
+        <p>
+          At these crucial times,it is always better to be familiar about the
+          interview process and an interactive way to help you perform better in
+          it is by engaging in mock interviews helping you identify your
+          strengths and weaknesses.
+        </p>
+      </section>
+      <section className={`${styles.heroImageContainer}`}>
+        <img src="assets/HeroSectionImage.svg" alt="" />
```
Comment (shivanigangadharan) [inline src/components/HeroSection/HeroSection.jsx]: can give an appropriate alt message here.

**Review by shivanigangadharan — APPROVED** [review]
Code and UI of carousel both look good. Just a design suggestion - you can keep the size (height) of the carousel a little bit smallesr as it seemed to be overflowing.
Overall nice work, can merge this. 👍🏼

File: src/components/HeroSection/HeroSection.jsx (hunk 3 above)
Comment (dev-enforced) [reply src/components/HeroSection/HeroSection.jsx:28]: Yes it is a dummy category so kept it empty for the time being!!

File: src/components/Categories/Categories.module.css (hunk 2 above)
Comment (dev-enforced) [reply src/components/Categories/Categories.module.css:39]: Yes this section will be edited once I get the categories by making a request to the backend in the upcoming PRs

File: src/App.module.css (hunk 1 above)
Comment (dev-enforced) [reply src/App.module.css:45]: Yes kept it for the purpose of other pages setup so would adjust it accordingly

File: src/components/HeroSection/HeroSection.jsx (hunk 3 above)
Comment (dev-enforced) [reply src/components/HeroSection/HeroSection.jsx]: Yes have noticed the change. Made the changes!!

**Pull request merged**